File: web/src/model/Game.js
const Game = (name, ageMin, ageMax, playersMin, playersMax, duration, imgURL) => {
  const initValue = "";
  const setInitValue = (value) => (value === undefined ? initValue : value);
  return {
    name: setInitValue(name),
    ageMin: setInitValue(ageMin),
    ageMax: setInitValue(ageMax),
    playersMin: setInitValue(playersMin),
    playersMax: setInitValue(playersMax),
    duration: setInitValue(duration),
    imgURL: setInitValue(imgURL)
  };
};

export default Game;

File: rest/src/main/java/com/challenge/rest/entity/Game.java
package com.challenge.rest.entity;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Data
@Table(name = "GAMES")
public class Game {

    @Id
    @Column(name = "ID")
    private int id;
    @Column(name = "NAME")
    private String name;
    @Column(name = "AGEMIN")
    private int ageMin;
    @Column(name = "AGEMAX")
    private int ageMax;
    @Column(name = "PLAYERSMIN")
    private int playersMin;
    @Column(name = "PLAYERSMAX")
    private int playersMax;
    @Column(name = "DURATION")
    private int duration;

}
